src/components/modals/RoleModal.vue

```diff
@@ -33,6 +56,9 @@ export default {
   components: { Token, Modal },
   props: ["playerIndex"],
   computed: {
+    extraTravellers() {
+      return [...this.$store.state.extraTravellers.values()];
```

Comment:
If you're already creating a helper function just for calling `.values()`, you might as well filter out the currently assigned player role, if it is part of the extraTravellers list, like what happens in `availableRoles`. Or, alternatively, add `extraTravellers` to `mapState` and then iterate over `extraTravellers.values()` directly in the template.